sql/init.sql

```diff
@@ -0,0 +1,58 @@
+--Neat trick to get these: dump dataframe to sql, then use the
+--sqlite3 CLI and call .schema tablename
+
+DROP TABLE IF EXISTS fpn;
+DROP TABLE IF EXISTS boal;
+
+CREATE TABLE fpn (
```

Comment:
Could do this wil sqlalamcy, then its slightly easier to import and work with the data - but perhaps that can be later on

Reply:
This was actually quite easy because I initially dumped the dataframe to the DB directly from pandas, then copied the create table commands that had been executed here so I could add the primary key constraints!